app/routes/bundesIdent/primaryoption.tsx

```diff
@@ -97,7 +88,7 @@ export default function BundesIdentVoraussetzung() {
         <Button
           className="w-full lg:max-w-[267px]"
           look="tertiary"
-          to="/identifikation"
+          to="/identifikation?origin=primaryoption"
```

Comment:
FYI: this query param is required to distinguish which calls should be redirected or not, not to be trapped in an infinite loop

```js
  if (
    !isPrimaryOptionRequest &&
    !isDropoutSurveyRequest &&
    shouldShowPrimaryOption
  ) {
    return redirect("/bundesIdent/primaryoption");
  }
```

Reply:
I am unsure whether this would change Plausible behavior

Reply:
> I am unsure whether this would change Plausible behavior 

I think this should not change plausible, because [plausible strips alls query parameters](https://plausible.io/docs/custom-query-params):
> By default, Plausible strips all query parameters for privacy purposes.